_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -275,12 +283,53 @@ class HealthCareApp extends React.Component {
           external: this.getExternalData(this.state.applicationData, statePath)
         });
       });
+    }
 
-      if (this.props.location.pathname === '/review-and-submit') {
-        buttons = (<SubmitButton onButtonClick={this.handleSubmit}/>);
-      } else {
-        buttons = (<ContinueButton onButtonClick={this.handleContinue}/>);
-      }
+    // Check which section the user is on and render the correct ProgressButtons.
+    const lastSectionText = (this.getUrl('back')) ? this.getUrl('back').split('/').slice(-1)[0].replace('-', ' ') : '';
+    const nextSectionText = (this.getUrl('next')) ? this.getUrl('next').split('/').slice(-1)[0].replace('-', ' ') : '';
+
+    const backButton = (
+      <ProgressButton
+          onButtonClick={this.handleBack}
+          buttonText={`Back to ${lastSectionText}`}
+          buttonClass={'usa-button-primary-alt'}/>
```

Comment:
Good question... had to remember where they were! https://github.com/department-of-veterans-affairs/healthcare-application/issues/25

Looks like the secondary button has a transparent background and a gray border. However you think it looks best is fine to do!